### PR TITLE
Clear card cache and fire UserChanged in SetTeamMemberShowcase

### PR DESCRIPTION
**Pull request by zapu**
cc @cjb

### CODE REVIEW

File: go/teams/showcase_helper.go [hunk 1]
```diff
@@ -111,5 +111,15 @@ func SetTeamMemberShowcase(ctx context.Context, g *libkb.GlobalContext, teamname
...
 	arg.Args.Add("is_showcased", libkb.B{Val: isShowcased})
 	_, err = g.API.Post(arg)
-	return err
+	if err != nil {
+		return err
+	}
+
+	// Clear usercard cache so when user goes to People tab,
+	// fresh card will be loaded.
+	u := g.LoginState().GetUID()
```
Comment (patrickxb) [inline go/teams/showcase_helper.go]: Can you change this to 

    u := g.ActiveDevice.UID()

Faster, less chance of failing...

File: go/teams/showcase_helper.go [hunk 2]
```diff
@@ -111,5 +111,15 @@ func SetTeamMemberShowcase(ctx context.Context, g *libkb.GlobalContext, teamname
...
+	g.Log.CDebugf(ctx, "Clearing Card cache for %s", u)
+	g.CardCache.Delete(u)
```
Comment (patrickxb) [inline go/teams/showcase_helper.go]: Can you log the error returned here if not nil?

File: go/teams/showcase_helper.go (hunk 1 above)
Comment (zapu) [reply go/teams/showcase_helper.go]: Should we change other occurrences as well?

Comment (patrickxb) [reply go/teams/showcase_helper.go]: If there are any others, yes.  Make a ticket?

The other option is `g.Env.GetUID()`, but I prefer the ActiveDevice one as it ensures that it is the uid of a provisioned user.

Comment (zapu) [reply go/teams/showcase_helper.go]: There is just one: https://github.com/keybase/client/blob/master/go/engine/profile_edit.go#L38-L40

this is where I copied this code from. 3 lines didn't seem worth factoring out but I can do that as well if preferable

Comment (patrickxb) [reply go/teams/showcase_helper.go]: Interesting.  I say go ahead and change that one too.  

I'll make a ticket to delete GetUID() in LoginState...perhaps there is some forgotten reason for it, but I can't think of anything.

**Pull request merged**